test/check-application

```diff
@@ -682,6 +682,32 @@ class TestFiles(testlib.MachineCase):
         self.assertEqual(m.execute("ls -l /home/admin/newfile")[:10], "-rwxrwxrwx")
         wait_permissions("Read, write and execute")
 
+        # As normal user you cannot change user/group permissions
+        b.drop_superuser()
+
+        m.execute("touch /home/admin/adminfile; chown admin: /home/admin/adminfile")
+        b.click("[data-item='adminfile']")
+        b.click("button:contains('Edit permissions')")
+        # A user cannot change ownership
+        b.wait_not_in_text(".pf-v5-c-modal-box__body", "Ownership")
```

Comment:
This is (at least potentially) a race condition -- you first need to wait on something positive in the form, like e.g. the permission input. Otherwise this would match a totally empty dialog while it's being rendered.

The easiest fix would be to just do the select_access() first.

Reply:
Hmm true! Maybe I should also leave the useEffect out then in the Dialog.